src/main/java/com/google/devtools/build/lib/remote/OperationNotFoundException.java

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.devtools.build.lib.remote;
+
+import java.io.IOException;
+
+/**
+ * An exception to indicate an operation loss.
+ */
```

Comment:
what specifically is an operation loss and why do we want to retry it?

Reply:
```
                        case DOES_NOT_EXIST:
                          // TODO(olaola): either make this retriable, or use a different exception.
                          throw new IOException(
                              String.format(
                                  "Operation %s lost on the remote server.", op.getName()));
```

The name was derived from the above, which it replaces. An operation is lost when a watch request responds with a change state of `DOES_NOT_EXIST`